test-util/src/main/java/io/camunda/zeebe/test/util/bpmn/random/blocks/ProcessBuilder.java

```diff
@@ -120,7 +122,23 @@ private void buildEventSubProcess(
   }
 
   public ExecutionPath findRandomExecutionPath(final Random random) {
-    final var followingPath = blockBuilder.findRandomExecutionPath(random);
+    final var startAnywhere = random.nextBoolean();
+
+    final ExecutionPathContext context;
+    final ExecutionPathSegment followingPath;
```

Comment:
:x: Can be moved outside the if statement. All that is needed is the context object initialized properly